src/matter/cluster/ScenesCluster.ts

```diff
@@ -0,0 +1,293 @@
+/**
+ * @license
+ * Copyright 2022 The node-matter Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import {
+    ArrayT,
+    BooleanT,
+    Field,
+    OptionalField,
+    ObjectT,
+    StringT,
+    UInt8T,
+    UInt16T,
+    UInt32T,
+    Bound,
+    EnumT, BitMapT, Bit, AnyT
+} from "../../codec/TlvObjectCodec";
+import { Attribute, OptionalAttribute, Cluster, Command, OptionalCommand, NoResponseT } from "./Cluster";
+import { StatusCode } from "../interaction/InteractionMessages";
+import { GroupIdT } from "../common/GroupId";
+import { ClusterIdT } from "../common/ClusterId";
+import { NodeIdT } from "../common/NodeId";
+import { AttributeIdT } from "../common/AttributeId";
+import { MatterApplicationClusterSpecificationV1_0 } from "../../Specifications";
+
+/** @see {@link MatterApplicationClusterSpecificationV1_0} § 1.4.6.1 */
+const AttributeValuePairT = ObjectT({
+    /**
+     * This field SHALL be present or not present, for all instances in the Scenes cluster. If this field is
+     * not present, then the data type of AttributeValue SHALL be determined by the order and data type defined
+     * in the cluster specification. Otherwise the data type of AttributeValue SHALL be the data type of the
+     * attribute indicated by AttributeID.
+     */
+    attributeId: Field(0, AttributeIdT),
+
+    /** This is the attribute value as part of an extension field set. */
+    attributeValue: Field(1, ArrayT(AnyT)),
+});
+
+/**
+ * This data type indicates for a given cluster a set of attributes and their values. Only attributes which
+ * have the "S" designation in the Quality column of the cluster specification SHALL be used in the
+ * AttributeValueList field.
+ *
+ * @see {@link MatterApplicationClusterSpecificationV1_0} § 1.4.6.2
+ */
+export const ExtensionFieldSetT = ObjectT({
+    clusterId: Field(0, ClusterIdT),
+    attributeValueList: Field(1, ArrayT(AttributeValuePairT)),
+});
+
+/** @see {@link MatterApplicationClusterSpecificationV1_0} § 1.4.9.2 */
+/** @see {@link MatterApplicationClusterSpecificationV1_0} § 1.4.9.9 */
```

Comment:
I think there can only be one JsDoc per element to be valid.

```suggestion
/** @see {@link MatterApplicationClusterSpecificationV1_0} § 1.4.9.2 and 1.4.9.9 */
```